### PR TITLE
LPS-134036

### PR DESCRIPTION
**Pull request by brunoquei2105**

### CODE REVIEW

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/dao/search/AccountRoleRowChecker.java [hunk 1]
```diff
@@ -36,7 +38,13 @@ public boolean isDisabled(Object object) {
...
 		Role role = accountRoleDisplay.getRole();
 
-		if (role.getType() == RoleConstants.TYPE_ACCOUNT) {
+		AccountRole accountRole =
+			AccountRoleLocalServiceUtil.fetchAccountRoleByRoleId(
+				role.getRoleId());
+
+		if ((role.getType() == RoleConstants.TYPE_ACCOUNT) &&
+			(accountRole.getAccountEntryId() == 0)) {
+
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/dao/search/AccountRoleRowChecker.java:47]: We can simplify this by using the `AccountRoleConstants. isSharedRole` to check for global account roles.

**Review by pei-jung — CHANGES_REQUESTED** [review]
@brunoquei2105 good job on the fix! We can simplify the code by using the util class. Please see my comment. Thanks!

**Comment by brunoquei2105** [comment]
ci:test:sf

**Comment by brunoquei2105** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**